### PR TITLE
Fix #144

### LINKED ISSUE #144: Add discussion section about loyalty program
We say in the problem statement section that the effectiveness of the loyalty program is something we want to look into, however we dont. We should maybe add a discussion section, or state somewhere else, that that has not been accomplished

### PR DESCRIPTION
**Pull request by TexWO**
Not sure if we need to explain more in detail for not addressing the loyalty program

### CODE REVIEW

File: src/loyalty-program/index.tex [hunk 1]
```diff
@@ -0,0 +1,14 @@
+%textidote: ignore begin
+\section{Loyalty Program}\label{sec:loyalty-program}
+%textidote: ignore end
+
+As mentioned in the problem statement~\ref{subsec:problem-statement} we wanted to look into the effectiveness
```
Comment (SebastianMygind) [inline src/loyalty-program/index.tex]: we should not use words such as "we", as it is better to keep it more formal through "the authors" or avoiding making use of pronouns.

File: src/loyalty-program/index.tex [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+of the loyalty program because having a loyalty program is a great way to increase customer retention and
+engagement.
+Loyalty programs are widely recognized as a powerful tool for increasing customer loyalty which leads to more visits,
+boosting the overall business performance
+
+However, during the project, our focus shifted towards the development of the application and its core features,
```
Comment (SebastianMygind) [inline src/loyalty-program/index.tex]: try avoiding pronouns, here: "our"

File: src/loyalty-program/index.tex [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+which were critical for ensuring its functionality and success.
+As a result, we were unable to further investigate the loyalty program's effectiveness,
```
Comment (SebastianMygind) [inline src/loyalty-program/index.tex]: avoid using "we"

**Review by SebastianMygind — CHANGES_REQUESTED** [review]
Nice, but try to avoid using pronouns

**Review by Danar435 — APPROVED** [review]

**Review by SebastianMygind — APPROVED** [review]

**Pull request merged**